config.go

```diff
@@ -47,7 +47,8 @@ type Config struct {
 	RPCLimitMethod            string   `confkey:"rpclimitmethod" default:"first" validate:"enum=first,random"`
 	LoggerType                string   `confkey:"logger_type" default:"file"`
 	FactCacheTime             int      `confkey:"fact_cache_time" default:"300"`
-	SSLCipher                 string   `confkey:"ssl_cipher" default:"aes-256-cbc"`
+	CipherSuites              []string `confkey:"cipher_suites" type:"comma_split"`
+	ECCCurves				  []string `confkey:"ecc_curves" type:"comma_split"`
```

Comment:
indent fix pls

Reply:
adding it here...problematic, ruby code will do dumb things like die on unknown things, so this will make ruby code fail, hence the plugin.choria.awfulstuff. 

This whole thing needs a revisit but wow big impact on users, so other security settings i think is just `plugin.security`, lets move this there.

https://github.com/choria-io/go-config/blob/32872409688e8185dd58fc864c526c1ef45d3a62/choria.go#L79